tasks/managed/collect-atlas-params/collect-atlas-params.yaml

```diff
@@ -139,14 +139,16 @@ spec:
         elif [ "$atlasServer" = "stage" ]; then
             bombasticApiUrl="https://sbom.atlas.release.stage.devshift.net"
             ssoTokenUrl="https://auth.stage.redhat.com/auth/realms/EmployeeIDP/protocol/openid-connect/token"
-            secretName="atlas-staging-sso-secret"
-            retryAWSSecretName="atlas-retry-s3-staging-secret"
+            secretName=$(jq -r '.atlas."atlas-sso-secret-name" // "atlas-staging-sso-secret"' "$DATA_FILE")
+            retryAWSSecretName=$(jq -r '.atlas."atlas-retry-aws-secret-name" // "atlas-retry-s3-staging-secret"' \
```

Comment:
These should be added in the schema file.

Reply:
added to schema